### ISSUE TITLE
error in server rendering the router app example

### ISSUE THREAD

**Issue by Soweyln** [shakacode/react-webpack-rails-tutorial#283]
in case of someone had error when running the server rendering example,
there is an error in the code
should be
```javascript
const { location } = railsContext;
//instead of const location = { railsContext };
```
https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/app/bundles/comments/startup/ServerRouterApp.jsx#L14

Hope it can help someone in case~

**Comment by justin808** [comment]
Interesting! Can you please submit a PR so you get credit for that find! 

EXTRA credit if you can make CI fail without your fix, and pass with your fix.

**Comment by justin808** [comment]
Thanks @Soweyln. I'm fixing this.

**Comment by justin808** [comment]
Fixed in #286.
